### DISCUSSION TITLE
kibana visualization error

### DISCUSSION THREAD

**Discussion by d3athg0d** [telekom-security/tpotce#1255] [Q&A] [answered]
Hi,

i am getting this error on every honeypots visualizations
quite noob about how can i correct this?

![Screenshot 2023-01-26 220938](https://user-images.githubusercontent.com/78822442/214857244-292c8a09-71f8-4e8c-8dad-36ad81dd03ee.png)

**Comment by t3chn0m4g3** [comment, marked as answer]
#1254
